spec/bolt/node/winrm_spec.rb

```diff
@@ -113,519 +114,531 @@ def stub_winrm_to_raise(klass, message)
     end
   end
 
-  it "executes a command on a host", winrm: true do
-    expect(winrm.execute(command).stdout.string).to eq("#{user}\r\n")
+  context "connecting over SSL" do
+    it "executes a command on a host", winrm: true do
```

Comment:
How does file transfer work? I feel like we should also test that over SSL.

Reply:
Good call, I'll add a test for that.

Reply:
Updated.